transformers/modeling_bert.py

```diff
@@ -819,7 +819,7 @@ def tie_weights(self):
                                    self.bert.embeddings.word_embeddings)
 
     def forward(self, input_ids, attention_mask=None, token_type_ids=None, position_ids=None, head_mask=None,
-                masked_lm_labels=None, encoder_hidden_states=None, encoder_attention_mask=None):
+                masked_lm_labels=None, lm_labels=None, encoder_hidden_states=None, encoder_attention_mask=None):
```

Comment:
You should organize the order of the inputs so that the inputs to `self.bert` forward pass form a contiguous block in the inputs to `BertForMaskedLM`. This is necessary for good torch.jit support (see #1010) as it uses tuple as inputs instad of dictionaries. So just put all the labels (`masked_lm_labels` and `lm_labels`) at the end if you can.

Reply:
`masked_lm`labels` the last argument before, leaving it there. `lm_labels` moved to the end.